mautic/mautic
# [UX] Replace default buttons (btn-default) with btn-ghost

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

Dependencies:
- #13770 

This PR replaces btn-default buttons and links with btn-ghost styles, aiming to start defining a button hierarchy across the entire UI.
The Ghost style seemed to be the closest to what we already have now.






| Before (btn-default)                                | After (btn-ghost)
| -------------------------------------- | ---
| ![image](https://github.com/user-attachments/assets/78dec814-b14c-4223-a7ca-c84f0ebd87a0)                                        | ![image](https://github.com/user-attachments/assets/167d2496-a415-42ff-89e9-eb389ddbfed5)


| Before (btn-default)                                | After (btn-tertiary)
| -------------------------------------- | ---
|  ![image](https://github.com/user-attachments/assets/2f68b6c4-a891-469f-a77c-75639ad9c5a5)                                      | ![image](https://github.com/user-attachments/assets/8ce7ddcb-a260-4a23-8e5e-381f115f9c72)


| Before (btn-default)                                | After (btn-ghost btn-icon)
| -------------------------------------- | ---
| ![image](https://github.com/user-attachments/assets/9af03d0b-48c7-4d47-bddd-affb485a08ff)                                       | ![image](https://github.com/user-attachments/assets/5bdf2a1d-07fc-4b07-a44f-7a2e40fc71c0)


| Before (btn-default)                                | After (btn-)
| -------------------------------------- | ---
|                                        | 


---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Campaigns > New > See the Launch campaign builder button
3. Open Contacts > Look at the Anonymous toggle icon
4. Create a contact > Save & Close > Scroll to the Timeline and see the Export button
5. Check other places to ensure everything is okay
